### PR TITLE
#164798172 Users should be able to like and or dislike articles

### PR DESCRIPTION
**Pull request by dannylwe**
#### What does this PR do?
Enables users to like or dislike an article

#### Description of Task to be completed?
Users are able to offer sentiment to an article by liking or disliking the article

#### How should this be manually tested?
run `git fetch like-dislike-164798172`
run `git checkout like-dislike-164798172`
run `npm i`
run `npm run start:dev`

#### Any background context you want to provide?
This PR is still in progress but feel free to leave some feedback

#### What are the relevant pivotal tracker stories?
[#164798172](https://www.pivotaltracker.com/story/show/164798172)

### CODE REVIEW

File: .netlify/state.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"siteId": "fbb9e46e-437f-4c7a-abdb-6bff035def71"
+}
```
Comment (ezrogha) [inline .netlify/state.json]: What is the purpose of this file @dannylwe ?

File: src/actions/likeDislikeActions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import axios from '../helpers/axiosInstance';
+import { LIKE_AN_ARTICLE, DISLIKE_AN_ARTICLE, ERROR } from './types';
+
+export const likeAnArticle = slug => dispatch => axios
+  .post(`${process.env.baseURL}/articles/${slug}/like`, {
+    likes: true,
+  })
+  .then((response) => {
+    dispatch({ type: LIKE_AN_ARTICLE, payload: 1 });
+    console.log(response.data);
```
Comment (ezrogha) [inline src/actions/likeDislikeActions.js]: Please don't forget to remove these console logs when your PR is ready

File: src/actions/profileActions.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-restricted-globals */
```
Comment (ezrogha) [inline src/actions/profileActions.js]: Please remove this comment

File: src/components/Articles/ArticleDetail.jsx [hunk 4]
```diff
@@ -14,14 +14,30 @@ const ArticleDetail = ({ article }) => (
             <h1 className="card-title article-title ">{article.title}</h1>
             <div className="row">
               <div className="col-md-3">
-                <img src={avartaImage} className="img-circle" alt="user-icon" width="70px" height="70px" />
+                <img
+                  src={avartaImage}
+                  className="img-circle"
+                  alt="user-icon"
+                  width="70px"
+                  height="70px"
```
Comment (ezrogha) [inline src/components/Articles/ArticleDetail.jsx:37]: kindly push these attributes (width and height) to the css file

File: src/components/Articles/ArticleDetail.jsx [hunk 5]
```diff
@@ -78,6 +93,32 @@ const ArticleDetail = ({ article }) => (
                 <i className="fas fa-trash-alt" />
               </button>
             </div>
+            {/* this is where to put thumbs up/down */}
```
Comment (ezrogha) [inline src/components/Articles/ArticleDetail.jsx]: please remove this comment

File: src/components/Articles/DisplayArticles.jsx [hunk 6]
```diff
@@ -9,7 +9,10 @@ const DisplayArticles = ({ article, articleDate }) => (
   <div className="row">
     <div className="col-md-12">
       <div className="row">
-        <div className="card col-md-12" style={{ boxShadow: '5px 10px 18px #888888' }}>
+        <div
+          className="card col-md-12"
+          style={{ boxShadow: '5px 10px 18px #888888' }}
```
Comment (ezrogha) [inline src/components/Articles/DisplayArticles.jsx:14]: Move styles to the css file

File: src/components/Articles/article.scss [hunk 7]
```diff
@@ -171,6 +171,9 @@ div.ReactTags__suggestions ul li.ReactTags__activeSuggestion {
 .custom-model-dialog{
     padding-top: 240px;
```
Comment (ezrogha) [inline src/components/Articles/article.scss]: Use rem instead of px

**Review by ezrogha — DISMISSED** [review]
Good work @dannylwe,
Kindly work on this feedback

File: src/components/Articles/ArticleDetail.jsx (hunk 4 above)
Comment (dannylwe) [reply src/components/Articles/ArticleDetail.jsx:37]: @ezrogha not my work, reformatting

File: src/components/Articles/DisplayArticles.jsx (hunk 6 above)
Comment (dannylwe) [reply src/components/Articles/DisplayArticles.jsx:14]: @ezrogha not my work, reformatting

File: src/components/Articles/article.scss (hunk 7 above)
Comment (dannylwe) [reply src/components/Articles/article.scss]: @ezrogha not my work, reformatting

File: .netlify/state.json (hunk 1 above)
Comment (felixkiryowa) [reply .netlify/state.json]: @dannylwe , you  should add a new line here

**Review by felixkiryowa — CHANGES_REQUESTED** [review]

File: src/actions/profileActions.js (hunk 3 above)
Comment (dannylwe) [reply src/actions/profileActions.js]: @ezrogha necessary to remove eslint error. The rule is off in eslint but the error is persisting.

File: .netlify/state.json (hunk 1 above)
Comment (dannylwe) [reply .netlify/state.json]: @felixkiryowa deleted file, not needed

File: src/actions/likeDislikeActions.js (hunk 2 above)
Comment (dannylwe) [reply src/actions/likeDislikeActions.js]: @ezrogha worked on feedback

**Review by felixkiryowa — DISMISSED** [review]
@dannylwe Rename your branch to follow convention

**Comment by dannylwe** [comment]
@felixkiryowa I noticed just now. Will have to close this PR and open a new one.

**Review by dorothyas — APPROVED** [review]
@dannylwe good work

**Review by hadijahkyampeire — APPROVED** [review]

**Pull request merged**